### DISCUSSION TITLE
Shortcode for multi day event

### DISCUSSION THREAD

**Discussion by CityFlake** [add2cal/add-to-calendar-button#406] [Q&A] [answered]
Hi, I'd like to setup a Multi-Date Event with shortcode. I didn't see another post on this. Could someone give me guidance or point me in the right direction? Thanks so much in advance!

**Edit: I am able to make it work with simple events. I just don't know how to create an array with the dates function.**

This is the sample code from the example page of the website that I'd like to convert to shortcode.

<add-to-calendar-button
  name="Event Series"
  dates='[
    {
      "name":"Reminder 1/3 to test the Add to Calendar Button",
      "description":"This is the first part to check the Add to Calendar Button script at [url]https://add-to-calendar-button.com/[/url]",
      "startDate":"today+3",
      "startTime":"10:15",
      "endTime":"23:30"
    },
    {
      "name":"Reminder 3/3 to test the Add to Calendar Button",
      "description":"This is the third part to check the Add to Calendar Button script at [url]https://add-to-calendar-button.com/[/url]",
      "startDate":"today+8",
      "startTime":"09:00",
      "endTime":"19:00"
    },
    {
      "name":"Reminder 2/3 to test the Add to Calendar Button",
      "description":"This is the second part to check the Add to Calendar Button script at [url]https://add-to-calendar-button.com/[/url]",
      "startDate":"today+5",
      "startTime":"11:30",
      "endTime":"20:00"
    }
  ]'
  timeZone="America/Los_Angeles"
  location="World Wide Web"
  options="'Apple','Google','iCal','Outlook.com','Yahoo'"
  lightMode="bodyScheme"
></add-to-calendar-button>

**Comment by jekuer** [comment, marked as answer]
The WordPress shortcode gets broken when you use square brackets!

To avoid them, you can omit the outer ones (directly go with dates='{...). If you need to use the url pattern for the description, you can use curly brackets there as an alternative - like {url}.

**Reply by CityFlake** [reply]
Thank you! Yes, this works! Super awesome plugin! Thanks for the support!

**Reply by jekuer** [reply]
Feel free to leave a review at the WordPress page. 😉
